2001-3000/LOC2531DaqqiI.xml

```diff
@@ -48,11 +48,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPlace>
             <place type="monastery">
-               <placeName>Däqqi Ita</placeName>
-               <placeName type="alt">Märagwǝz</placeName>
-               <placeName type="alt">Däbrä Däqqi Ita</placeName>
+               <placeName xml:lang="ti" xml:id="n1">ደቂ፡ ኢታ፡</placeName>
+               <placeName xml:lang="ti" corresp="#n1" type="normalized">Daqqi ʾItā</placeName>
+               <placeName type="alt">Marāgʷǝz</placeName>
+               <placeName type="alt">Dabra Daqqi ʾItā</placeName>
+               <placeName type="alt">Gadam Daqqi ʾItā</placeName>
             </place>
```

Comment:
add 
```
<country ref="LOC2991Eritre"/>
               <region ref="..."/>      
<settlement ref="..."/>
```

Reply:
Actually, I do not know why Marāgʷǝz stands here. Daqqi Ita can be/ probably is the same as  ደቂ ጣዒስ, no idea what the difference is so significant. I will add it.

Reply:
Ok, but not here, to Daqqi Ita.